CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to the Zowe API Mediation Layer package will be documented in this file.
 
+## `1.12.0`
+
+- Provide Zowe Authentication and Authorization Service (ZAAS) client
+- Refresh the static client definitions from the API Catalog UI
+- Switch to sso-auth instead of apiml-auth
+- Added logout endpoint api doc
+- Make jjwt only a test dependency
+- Fix order of fetching JWT from request
+
```

Comment:
- Implement request retrying for service instances

Reply:
Added